samples/net/mqtt_publisher/src/config.h

```diff
@@ -21,7 +21,11 @@
 #define SERVER_ADDR		"2001:db8::2"
 #else
 #define ZEPHYR_ADDR		"192.168.1.101"
-#define SERVER_ADDR		"192.168.1.10"
+#if (defined(CONFIG_NET_SOCKETS_OFFLOAD) && !defined(CONFIG_MBEDTLS))
```

Comment:
I think `!defined(CONFIG_MBEDTLS)` is not needed here. Again, using Zephyr's secure socket with offloaded sockets is not a valid configuration, it's the same API.

Reply:
Thanks for the clarification. That was the part I didn't know. I will remove this check in general.